ui/hooks/common.ts

```diff
@@ -22,7 +22,7 @@ export function useRequestState<T>(): [
 ] {
   const [state, setState] = useState<RequestState<T>>({
     value: null,
-    loading: true,
+    loading: false,
```

Comment:
Just for my edification, why did this change?

Reply:
Setting `loading` to true when the hook is instantiated isn't really accurate. Loading should get set when the request gets started.